### ISSUE TITLE
Is is possible to use losses like mmi and mwer in k2 to finetune a model from another framework

### ISSUE THREAD

**Issue by 1215thebqtic** [k2-fsa/icefall#793]
Hi,

Is is possible to use losses like mmi and mwer in k2 to finetune a model from another framework like wenet? Training from scratch using icefall may take a lot of time, so I'm considering using k2 to finetune a pre-trained model from wenet first. Thanks.

**Comment by danpovey** [comment]
It may be possible, but it would take deep knowledge of k2.
So far I believe we have not been able to find a scenario where this kind of fine-tuning (with a one of those types of loss) would be helpful.

**Comment by desh2608** [comment]
One possible scenario where MWER may help is for long-form ASR, i.e., when the eval-time utterances are significantly longer than those seen during training. Check this paper from Google for example: https://arxiv.org/abs/2110.03841
